ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -287,6 +287,11 @@ data instance BlockQuery (ShelleyBlock proto era) :: Type -> Type where
   GetAccountState
     :: BlockQuery (ShelleyBlock proto era) AccountState
 
+  GetSPOStakeDistr
+    :: CG.ConwayEraGov era
+    => Set (KeyHash 'StakePool (EraCrypto era))
+    -> BlockQuery (ShelleyBlock proto era) (Map (KeyHash 'StakePool (EraCrypto era)) Coin)
```

Comment:
Good point. However, I can't push anymore for some reason. I get a permission denied error. Could it be because I force-pushed from GitHub to bring this branch up-to-date with `main`?